src/runtime_src/xocl/api/clReleaseCommandQueue.cpp

```diff
@@ -37,7 +37,7 @@ static cl_int
 clReleaseCommandQueue(cl_command_queue command_queue)
 {
   validOrError(command_queue);
-  if (xocl(command_queue)->release())
+  if (xocl(command_queue) != nullptr && xocl(command_queue)->release())
```

Comment:
validOrError checks this, for better or worse it is an OpenCL implementation idiom to not have error checks outside of validOrError.  Please revert.